test/cmd/apply.sh

```diff
@@ -295,7 +295,7 @@ __EOF__
   kube::test::get_object_assert 'pods pod-c' "{{${id_field}}}" 'pod-c'
   # cleanup
   kubectl delete pod pod-a pod-c "${kube_flags[@]:?}"
-  kube::test::get_object_assert pods "{{range.items}}{{${id_field:?}}}:{{end}}" ''
+  kube::test::wait_object_assert pods "{{range.items}}{{${id_field:?}}}:{{end}}" ''
```

Comment:
`kubectl delete pods` waits for pods to actually be deleted by default before returning... did you actually see flakes here?

Reply:
No, I was overzealous. So, I can get rid of these. The one above is a real flake.

Reply:
I have updated the PR, to just fix this flaky test.